Capture-Raw-Video-Data/Agora-Media-Source-Windows/AgoraMediaSource/AgoraObject.cpp

```diff
@@ -394,10 +394,11 @@ BOOL CAgoraObject::EnableEchoTest(BOOL bEnable)
 
 BOOL CAgoraObject::SetVideoProfileEx(int nWidth, int nHeight, int nFrameRate, int nBitRate)
 {
-	IRtcEngine2 *lpRtcEngine2 = (IRtcEngine2 *)m_lpAgoraEngine;
-	int nRet = lpRtcEngine2->setVideoProfileEx(nWidth, nHeight, nFrameRate, nBitRate);
+	//IRtcEngine2 *lpRtcEngine2 = (IRtcEngine2 *)m_lpAgoraEngine;

```

Comment:
can we delete the whole setVideoProfileEx function? if it's not useful any more

Reply:
setVideoProfileEx should call setVideoEncoderConfiguration. I'll make a new pr.